website/intro/index.html.markdown

```diff
@@ -3,79 +3,42 @@ layout: "intro"
+  Learn what Terraform is, what problems it can solve, and how it compares to existing software.
 ---
 
 # Introduction to Terraform
 
-Welcome to the intro guide to Terraform! This guide is the best
-place to start with Terraform. We cover what Terraform is, what
-problems it can solve, how it compares to existing software,
-and contains a quick start for using Terraform.
+Terraform is an infrastructure as code (IaC) tool that allows you to build, change, and version infrastructure safely and efficiently. This includes low-level components such as compute instances, storage, and networking, as well as high-level components such as DNS entries, SaaS features, etc. Terraform can manage both existing service providers and custom in-house solutions. 
 
-If you are already familiar with the basics of Terraform, the
-[documentation](/docs/index.html) provides a better reference
-guide for all available features as well as internals.
-
-> **Hands-on:** Try the [Get Started](https://learn.hashicorp.com/tutorials/terraform/infrastructure-as-code) tutorials on HashiCorp Learn.
-
-## What is Terraform?
-
-Terraform is a tool for building, changing, and versioning infrastructure
-safely and efficiently. Terraform can manage existing and popular service
-providers as well as custom in-house solutions.
-
-Configuration files describe to Terraform the components needed to
-run a single application or your entire datacenter.
-Terraform generates an execution plan describing
-what it will do to reach the desired state, and then executes it to build the
-described infrastructure. As the configuration changes, Terraform is able
-to determine what changed and create incremental execution plans which
-can be applied.
-
-The infrastructure Terraform can manage includes
-low-level components such as
-compute instances, storage, and networking, as well as high-level
-components such as DNS entries, SaaS features, etc.
+Below, HashiCorp co-founder and CTO Armon Dadgar describes how Terraform can help solve common infrastructure challenges.
 
 <iframe src="https://www.youtube.com/embed/h970ZBgKINg" frameborder="0" allowfullscreen="true"  width="560" height="315" ></iframe>
 
-Examples work best to showcase Terraform. Please see the
-[use cases](/intro/use-cases.html).
 
-The key features of Terraform are:
+
+## Key Features
 
 ### Infrastructure as Code
 
-Infrastructure is described using a high-level configuration syntax. This allows
-a blueprint of your datacenter to be versioned and treated as you would any
-other code. Additionally, infrastructure can be shared and re-used.
+Infrastructure is described using a high-level [configuration language](/docs/language/index.html) in human-readable, declarative configuration files. This allows you to create a blueprint that can be versioned, shared, and reused.
```

Comment:
Active vice 

- is described
- can be versioned ...

Reply:
Thank you! fixed in my recent commit!